From 71dcd92da3e21019bd224ca4c13aeb98109614eb Mon Sep 17 00:00:00 2001
From: Alan Coopersmith <alan.coopersmith@oracle.com>
Date: Sat, 2 Jan 2016 19:50:30 -0800
Subject: [PATCH] GNOME desktop menu

Changes needed for integration into GNOME desktop menus.
Localized name & tooltips (Sun bug 5103358 / Oracle bug 15227916)

Unknown why this differs from upstream - need to find out someday.
---
 driver/screensaver-properties.desktop.in | 34 ++++++++++++++++++++++++++++----
 1 file changed, 30 insertions(+), 4 deletions(-)

diff --git a/driver/screensaver-properties.desktop.in b/driver/screensaver-properties.desktop.in
index de42527..9b9dbe0 100644
--- a/driver/screensaver-properties.desktop.in
+++ b/driver/screensaver-properties.desktop.in
@@ -1,8 +1,34 @@
 [Desktop Entry]
 Exec=xscreensaver-demo
 Icon=xscreensaver
 Terminal=false
-_Name=Screensaver
-_Comment=Change screensaver properties
+Name=Screensaver
+Name[cs]=Spořič obrazovky
+Name[de]=Bildschirmschoner
+Name[es]=Salvapantallas
+Name[fr]=Économiseur d'écran
+Name[hu]=Képernyővédő
+Name[it]=Salvaschermo
+Name[ja]=スクリーンセーバー
+Name[ko]=화면 보호기
+Name[pt_BR]=Protetor de tela
+Name[sv]=Skärmsläckare
+Name[zh_CN]=屏幕保护程序
+Name[zh_HK]=螢幕保護程式
+Name[zh_TW]=螢幕保護程式
+Comment=Choose screensaver delay and appearance.
+Comment[cs]=Konfigurace nastavení spořiče obrazovky.
+Comment[de]=Bildschirmschonereinstellungen konfigurieren
+Comment[es]=Configura los valores del salvapantallas.
+Comment[fr]=Configurer les paramètres de l'économiseur d'écran.
+Comment[hu]=A képernyővédő beállításainak megváltoztatása
+Comment[it]=Configura le impostazioni del salvaschermo.
+Comment[ja]=スクリーンセーバーを設定
+Comment[ko]=화면 보호기 설정 구성
+Comment[pt_BR]=Definir as configurações do protetor de tela.
+Comment[sv]=Konfigurera inställningarna för skärmsläckaren.
+Comment[zh_CN]=配置屏幕保护程序的设置。
+Comment[zh_HK]=配置螢幕保護程式設定。
+Comment[zh_TW]=配置螢幕保護程式設定。
 Type=Application
 Categories=Settings;DesktopSettings;Security;X-XFCE;
-- 
2.6.1

